### ISSUE TITLE
I updated the OS of my ti nspire cx calculator 

### ISSUE THREAD

**Issue by Block-Flock** [ndless-nspire/Ndless#378]
I think before it was 3.6 but then I clicked some press to test thing and followed some tutorial to fix that and clicked send os and now im stuck with an os that I cant jailbreak which is 4.5.5 so is there any way for me to get this fixed?

**Comment by Oh64** [comment]
If you are on 4.5 just update to 5.2 or 5.4 and you can install Ndless.

**Comment by adriweb** [comment]
> If you are on 4.5 just update to 5.2 or 5.3 and you can install Ndless.

lol no 4.x are for CX models and 5.x and later for CX II models, exclusively.

**Comment by Vogtinator** [comment]
With a serial cable you might be able to install nLoader and downgrade that way.

**Comment by Block-Flock** [comment]
> With a serial cable you might be able to install nLoader and downgrade that way.

Yeah maybe, I also found out I can delete the OS for the nspire by pressing some buttons

**Comment by Vogtinator** [comment]
That will not reset the downgrade protection.

**Comment by Block-Flock** [comment]
> That will not reset the downgrade protection.

Is there any possible way i could do something or am i just stuck like this

**Comment by Oh64** [comment]
Oh sorry I didn't know.

**Comment by Block-Flock** [comment]
> Oh sorry I didn't know.
> So you have to... Wait.

Dang, I mean i looked at something on here where basically they editted the boot file or something and downgraded it

**Comment by Oh64** [comment]
I've been looking for a way to do this but even after testing things I still haven't found it.

**Comment by Block-Flock** [comment]
> With a serial cable you might be able to install nLoader and downgrade that way.

wait I have a usb cable that I plug into my calculator to connect it to my computer, do you think that nLoader can do anything for me or still no because i dont have ndless

**Comment by Isthecodebroken** [comment]
> With a serial cable you might be able to install nLoader and downgrade that way.

How exactly? With what software?

**Comment by Vogtinator** [comment]
https://github.com/parrotgeek1/TI-Nspire/tree/main/nLoader

**Comment by YxLit** [comment]
Thanks, but just out of curiosity, do you have an estimate for when endless 4.5.5.79 is coming out?

**Comment by YxLit** [comment]
Oh yeah, I have a serial cable but it's an RS232 male and my PC has a Male port for it too and I don't have a RS232 female to female changer to plug it in my PC. Anything else that I can try? Or do I wait for Ndless 4.5.5.79?

**Comment by Block-Flock** [comment]
> Oh yeah, I have a serial cable but it's an RS232 male and my PC has a Male port for it too and I don't have a RS232 female to female wire to plug it in my PC. Anything else that I can try? Or do I wait for Ndless 4.5.5.79?

I mean I just use a mini usb ig i could try it out

**Comment by SozinFriend** [comment]
yooo please upgrade ndless for 5.4 dang it

**Comment by Lockl00p** [comment]
> yooo please upgrade ndless for 5.4 dang it

While I do need this, I understand that the entirety of ndless is just finding an exploit that works, and thus could take a year

**Comment by PinkHatGithub** [comment]
> yooo please upgrade ndless for 5.4 dang it

We've said this so many times before. We don't know when it's going to be, We do know that fabian is still working on it. stop re-opening duplicates of #300. we have enough.

Sorry if I seemed a bit annoyed, I just have so many emails of the same things.

**Comment by sasdallas** [comment]
> yooo please upgrade ndless for 5.4 dang it

@adriweb Resolved
